portality/templates/application_form/editorial_form_body.html

```diff
@@ -3,12 +3,10 @@ <h1>{{ obj.bibjson().title }}</h1>
 {% include "application_form/_backend_validation.html" %}
 {% include "application_form/_contact.html" %}
 
-{% if obj["es_type"] == "application" and obj.application_type == "update_request" and (not current_journal or not current_journal.is_in_doaj()) %}
-    <h2 class="alert">
-        <span data-feather="alert-triangle" aria-hidden="true"></span>
-        Journal has been withdrawn or deleted. This update request was automatically rejected.
-    </h2>
-{% endif %}
+{% set related_journal = obj.related_journal_object %}
```

Comment:
this variable `related_journal` seem like never used

Reply:
yup, good spot, must have been left over from the merge